memo.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'sinatra/reloader'
+require 'json'
+require 'cgi'
+
+FILE_PATH = 'public/memos.json'
+
+def get_memos(file_path)
+  File.open(file_path) { |f| JSON.parse(f.read) }
+end
+
+def set_memos(file_path, memos)
+  File.open(file_path, 'w') { |f| JSON.dump(memos, f) }
+end
+
+get '/' do
+  redirect '/memos'
+end
+
+get '/memos' do
+  @memos = get_memos(FILE_PATH)
+  erb :index
+end
+
+get '/memos/new' do
+  erb :new
+end
+
+post '/memos' do
+  @title = params[:title]
+  @content = params[:content]
+
+  memos = get_memos(FILE_PATH)
+  maxid = 0
+  memos.each_key do |id|
+    maxid = id.to_i if id.to_i > maxid
+  end
+  id = (maxid + 1).to_s
```

Comment:
```suggestion
id = memos.keys.map(&:to_i).max + 1
```

修正する必要はありませんが、IDの最大値を探してインクリメントするなら、こんな風にも書けますよ 😄

Reply:
ご確認いただきありがとうございます！
修正しました。([28550eb](https://github.com/kazunoko1606/sinatra-memo-app/pull/1/commits/28550eb701dd5b53f035d5531ec31d03be12485c))
宜しくお願い致します。